.storybook/preview.jsx

```diff
@@ -7,7 +7,6 @@ export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
   controls: {
     matchers: {
-      color: /(background|color)$/i,
```

Comment:
this ended up colliding with us wanting to limit colors to our own tokens, so needed to remove this